apps/api/eslint.config.mjs

```diff
@@ -0,0 +1,29 @@
+import path from 'node:path'
+import { fileURLToPath } from 'node:url'
+
+import { FlatCompat } from '@eslint/eslintrc'
+import js from '@eslint/js'
+import globals from 'globals'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all,
+})
+
+export default [
+  ...compat.extends('./node_modules/@latitude-data/eslint-config/library.js'),
+  {
+    languageOptions: {
+      globals: {
+        ...globals.node,
+        ...Object.fromEntries(
+          Object.entries(globals.browser).map(([key]) => [key, 'off']),
+        ),
+        console: true,
+      },
+    },
+  },
+]
```

Comment:
i've updated to eslint 9 which deprecates .eslintrc files in favor of these monstruosities (that area automatically generated)

Reply:
Wow